etc/DependencyInstaller.sh

```diff
@@ -341,12 +355,14 @@ EOF
         _installUbuntuRuntime "${version}"
         if [[ "${option}" == "dev" ]]; then
             _installUbuntuDev
-            _installCommonDev "${version}"
+            _installCommonDev
```

Comment:
Is "${version}" no longer needed?

Reply:
not here, only for `_installUbuntuRuntime()`